<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FightManager</class>
 <widget class="QMainWindow" name="FightManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>711</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Fight</string>
    </property>
    <widget class="QProgressBar" name="hp1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift SemiBold</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
border-radius: 1px;
   background-color: #FFFFFF;
	text-align: center;

}
 
QProgressBar::chunk {
   background-color: #ff0000;

}</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>48</number>
     </property>
     <property name="format">
      <string>%v</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="hp2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift SemiBold</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
border-radius: 1px;
   background-color: #FFFFFF;
	text-align: center;

}
 
QProgressBar::chunk {
   background-color: #ff0000;

}</string>
     </property>
     <property name="value">
      <number>20</number>
     </property>
     <property name="format">
      <string>%v</string>
     </property>
    </widget>
    <widget class="QLabel" name="AT_pic">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>131</width>
       <height>201</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="DE_pic">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>50</y>
       <width>131</width>
       <height>201</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>70</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 9pt &quot;Adobe Garamond Pro Bold&quot;;
font: 75 30pt &quot;Adobe Caslon Pro Bold&quot;;</string>
     </property>
     <property name="text">
      <string>VS</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="text_fight">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>581</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="repeat_fight">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>580</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>重新开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>580</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>580</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>退出登录</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
